rest-api-spec/src/main/resources/rest-api-spec/api/tasks.cancel.json

```diff
@@ -4,18 +4,18 @@
     "methods": ["POST"],
     "url": {
       "path": "/_tasks",
-      "paths": ["/_tasks/_cancel", "/_tasks/{node_id}/_cancel", "/_tasks/{node_id}/{task_id}/_cancel"],
+      "paths": ["/_tasks/_cancel", "/_tasks/{task_id}/_cancel"],
       "parts": {
-        "node_id": {
-          "type": "list",
-          "description": "A comma-separated list of node IDs or names to limit the request; use `_local` to cancel only tasks on the node you're connecting to, leave empty to cancel tasks on all nodes"
-        },
         "task_id": {
           "type": "number",
           "description": "Cancel the task with specified id"
         }
       },
       "params": {
+        "node_id": {
+          "type": "list",
+          "description": "A comma-separated list of node IDs or names to limit the returned information; use `_local` to return information from the node you're connecting to, leave empty to get information from all nodes"
```

Comment:
I'm worried about nodes that are named "`prod:data:1`", won't that cause errors during the task id parsing?